src/libotutil/ot-tool-util.c

```diff
@@ -62,3 +62,55 @@ ot_parse_keyvalue (const char  *keyvalue,
   *out_value = g_strdup (eq + 1);
   return TRUE;
 }
+
+/**
+ * Note: A temporary function copied directly from 
+ * https://developer.gnome.org/glib/stable/glib-Pointer-Arrays.html#g-ptr-array-find-with-equal-func
```

Comment:
This is good to link to the documentation - could we also have a link to here? https://github.com/GNOME/glib/blob/a419146578a42c760cff684292465b38df855f75/glib/garray.c#L1664

Maybe something like:
```
 * "Note: temporarily copied from GLib: https://github.com/GNOME/glib/blob/a419146578a42c760cff684292465b38df855f75/glib/garray.c#L1664
 * See documentation at: https://developer.gnome.org/glib/stable/glib-Pointer-Arrays.html#g-ptr-array-find-with-equal-func
```